.travis.yml

```diff
@@ -2,18 +2,14 @@
 language: julia
 os:
   - linux
-  - osx
+  # - osx
```

Comment:
I think we should still test on osx, but allow failures.

Reply:
That's being done in GH Actions.